rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+max_width = 100
+use_field_init_shorthand = true
```

Comment:
Nice, I didn't know you could set these on stable. These seem like a simple improvement

I don't really like this being at the root, but that's also not avoidable